pkg/client/clientset/versioned/typed/configuration/v1/fake/fake_policy.go

```diff
@@ -86,6 +86,18 @@ func (c *FakePolicies) Update(ctx context.Context, policy *configurationv1.Polic
 	return obj.(*configurationv1.Policy), err
 }
 
+// UpdateStatus was generated because the type contains a Status member.
+// Add a +genclient:noStatus comment above the type to avoid generating UpdateStatus().
+func (c *FakePolicies) UpdateStatus(ctx context.Context, policy *configurationv1.Policy, opts v1.UpdateOptions) (*configurationv1.Policy, error) {
```

Comment:
Since this is generated code, it seems odd that it contains a reference to `FakePolicies`. Isn't this a test struct?

Reply:
yeah but this code is, as you said, generated by kubernetes `https://github.com/nginxinc/kubernetes-ingress/tree/master/vendor/k8s.io/code-generator` so i don't think we can control what it generates.